src/qt/forms/debugwindow.ui

```diff
@@ -1100,13 +1100,39 @@
                 </widget>
                </item>
                <item row="2" column="0">
+                <widget class="QLabel" name="peerNetworkLabel">
+                 <property name="toolTip">
+                  <string>The network protocol this peer is connected through.</string>
+                 </property>
```

Comment:
b1704ead912ed813eaa08e80af5e14a2c087574e

It is inconsistent to have a tooltip only for this label among others.

Reply:
There is one for the Mapped AS field as well. Probably best to add the missing ones but not in this pull.